geopandas/plotting.py

```diff
@@ -3,45 +3,42 @@
 import numpy as np
 import pandas as pd
 
+import geopandas as gpd
```

Comment:
small nit: can you do `from geopandas import GeoSeries` (that seems more consistent with other files; in any case we don't use `import geopandas as gpd` internally right now)

Reply:
That is not si simple, in fact. Doing `from geopandas import GeoSeries` raises error as `GeoSeries` imports `plot_series`, which imports `GeoSeries`...

```
ImportError while importing test module '/Users/martin/Dropbox/Python/geopandas/martinfleis/geopandas/tests/test_plotting.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../__init__.py:1: in <module>
    from geopandas.geoseries import GeoSeries  # noqa
../geoseries.py:16: in <module>
    from geopandas.plotting import plot_series
../plotting.py:6: in <module>
    from geopandas import GeoSeries
E   ImportError: cannot import name 'GeoSeries' from 'geopandas' (/Users/martin/Dropbox/Python/geopandas/martinfleis/geopandas/__init__.py)
```

Reply:
Huh, I would have expected both to be exactly the same regarding circular imports .. (also `import geopandas as gpd` does import GeoSeries in the init).

But, we should actually move the `geopandas.plotting` import into the `GeoSeries.plot` function to avoid this circular import. Or move the GeoSeries import here into the plot_.. function.